### PR TITLE
Update Docker environment and instructions

### PR DESCRIPTION
**Pull request by ppcano**
To make it work and consistent with https://github.com/grafana/xk6-output-timescaledb/

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,16 +1,15 @@
-FROM golang:1.17-alpine as builder
-RUN go install go.k6.io/xk6/cmd/xk6@latest
-RUN apk --no-cache add git
+FROM golang:1.16-alpine as builder
```
Comment (codebien) [inline Dockerfile:1]: I think we can keep the 1.17 version

File: Dockerfile [hunk 2]
```diff
@@ -1,16 +1,15 @@
...
 WORKDIR $GOPATH/src/go.k6.io/k6
 ADD . .
-RUN xk6 build --output /go/bin/k6 \
-	--with github.com/grafana/xk6-output-influxdb=.
+RUN apk --no-cache add git
+RUN CGO_ENABLED=0 go install go.k6.io/xk6/cmd/xk6@latest
+RUN CGO_ENABLED=0 xk6 build --with github.com/grafana/xk6-output-influxdb=. --output /tmp/k6
```
Comment (codebien) [inline Dockerfile:6]:
```suggestion
RUN go install go.k6.io/xk6/cmd/xk6@latest
RUN xk6 build --with github.com/grafana/xk6-output-influxdb=. --output /tmp/k6
```

File: Dockerfile [hunk 3]
```diff
@@ -1,16 +1,15 @@
...
 
-FROM alpine:3.14
+FROM alpine:3.13
```
Comment (codebien) [inline Dockerfile:8]:
```suggestion
FROM alpine:3.14
```

File: Dockerfile (hunk 2 above)
Comment (codebien) [inline Dockerfile:6]:
```suggestion
RUN CGO_ENABLED=0 xk6 build --with github.com/grafana/xk6-output-influxdb=.
```
Why not use the default folder?

**Review by codebien — CHANGES_REQUESTED** [review]
Few comments, thanks for it :pray:

File: Dockerfile (hunk 2 above)
Comment (ppcano) [reply Dockerfile:6]: @codebien  -> Line 11 -> `COPY --from=builder /tmp/k6 /usr/bin/k6`

Comment (codebien) [reply Dockerfile:6]: it is the motivation why the Line 11 before was `COPY --from=builder /go/bin/k6 /usr/bin/k6` for using the default Go binary folder

**Pull request merged**